website/versioned_docs/version-0.10.0/overview.md

```diff
@@ -13,7 +13,7 @@ how to learn more to get started.
 Apache Hudi (pronounced “hoodie”) is the next generation [streaming data lake platform](/blog/2021/07/21/streaming-data-lake-platform). 
 Apache Hudi brings core warehouse and database functionality directly to a data lake. Hudi provides [tables](/docs/next/table_management), 
 [transactions](/docs/next/timeline), [efficient upserts/deletes](/docs/next/write_operations), [advanced indexes](/docs/next/indexing), 
-[streaming ingestion services](/docs/next/hoodie_deltastreamer), data [clustering](/docs/next/clustering)/[compaction](/docs/next/compaction) optimizations, 
+[streaming ingestion services](/docs/next/hoodie_streamer), data [clustering](/docs/next/clustering)/[compaction](/docs/next/compaction) optimizations, 
```

Comment:
all of the older version should still link to the older deltastreamer pages without renaming, e.g., `/docs/0.10.0/hoodie_deltastreamer`.  Could you also try to see if no change works here?

Reply:
reverted hoodie_deltastreamer file rename change to avoid getting into such errors.